Yarn add is not working

# Bug Report


## Problem 
While running any yarn add command like `yarn add react-native-share@7.0.2`, it is showing error
```
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make: Entering directory '/home/parvesh_monu/Desktop/ArchFiles/React/status-react/build'
  CXX(target) Release/obj.target/status_nodejs_addon/modules/react-native-status/nodejs/status.o
../modules/react-native-status/nodejs/status.cpp:6:10: fatal error: ../../../result/libstatus.h: No such file or directory
    6 | #include "../../../result/libstatus.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [status_nodejs_addon.target.mk:107: Release/obj.target/status_nodejs_addon/modules/react-native-status/nodejs/status.o] Error 1
make: Leaving directory '/home/parvesh_monu/Desktop/ArchFiles/React/status-react/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Linux 5.18.1-arch1-1
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/parvesh_monu/Desktop/ArchFiles/React/status-react
gyp ERR! node -v v18.2.0
gyp ERR! node-gyp -v v9.0.0
gyp ERR! not ok 
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```


#### Expected behavior
`yarn add` should work


#### Actual behavior
`yarn add` not working


### Additional Information

- Status version: develop 
- Operating System: Linux

--- a/package.json
+++ b/package.json
@@ -6,13 +6,13 @@
   "scripts": {
     "start": "react-native start",
     "ios": "react-native run-ios",
+    "preinstall": "make status-go-library",
     "app:compile:android": "shadow-cljs compile android",
     "app:watch": "shadow-cljs watch android",
     "app:packager": "react-native start --host 0.0.0.0 --port 8081",
     "app:android": "react-native run-android"
   },
   "dependencies": {
-    "@mapbox/node-pre-gyp": "^1.0.9",
     "@react-native-community/async-storage": "^1.11.0",
     "@react-native-community/audio-toolkit": "git+https://github.com/tbenr/react-native-audio-toolkit.git#refs/tags/v2.0.3-status-v6",
     "@react-native-community/blur": "^3.6.0",
@@ -75,6 +75,7 @@
     "web3-utils": "^1.2.1"
   },
   "devDependencies": {
+    "@mapbox/node-pre-gyp": "^1.0.9",
     "@babel/generator": "7.0.0",
     "@babel/helper-builder-react-jsx": "7.0.0",
     "@babel/plugin-transform-block-scoping": "7.0.0",

